stdlib/README.md

```diff
@@ -0,0 +1,3 @@
+`libc.ll` is generated from a modified [musl](http://www.etalabs.net/musl/).
```

Comment:
I have no idea what you mean here.  As a potential external user of your semantics, I am confused.